### PR TITLE
adds secondary button style (red border)

### PR DESCRIPTION
**Pull request by stubbornella**
[Finishes #119301811]
- adjusts font-weight, font-size, padding on all buttons

This is the new button style @jerrygabra requested.
screen shot 2016-05-11 at 12 47 34 pm

### CODE REVIEW

File: src/pivotal-ui/components/pui-variables.scss [hunk 1]
```diff
@@ -461,8 +461,16 @@ $table-scrollable-body-border:   1px solid $gray-8;
 // Buttons
 // -------------------------
 
-$btn-font-weight:                600 !default;
+$btn-font-weight:                400 !default;
 $btn-border-width:               0;
+$btn-font-size:                  18px;
+
+$btn-sm-padding: 5px 20px 4px 20px;
```
Comment (aredridel) [inline src/pivotal-ui/components/pui-variables.scss]: Why a variable for this, but inline for `.btn` ?

File: src/pivotal-ui/components/pui-variables.scss [hunk 2]
```diff
@@ -461,8 +461,16 @@ $table-scrollable-body-border:   1px solid $gray-8;
 // Buttons
 // -------------------------
 
-$btn-font-weight:                600 !default;
+$btn-font-weight:                400 !default;
```
Comment (aredridel) [inline src/pivotal-ui/components/pui-variables.scss:464]: Does this change all buttons?

File: src/pivotal-ui/components/pui-variables.scss (hunk 1 above)
Comment (stubbornella) [inline src/pivotal-ui/components/pui-variables.scss]: This was actually already the case, the variables were just in the buttons.scss file. I moved them over here. It's a good point though. I'll update the PR to make both of them vars.

File: src/pivotal-ui/components/pui-variables.scss (hunk 2 above)
Comment (stubbornella) [inline src/pivotal-ui/components/pui-variables.scss:464]: Yes, that was intentional. Jerry felt they were too bold.

Comment (aredridel) [inline src/pivotal-ui/components/pui-variables.scss:464]: Cool. Just the kind of thing that'd be great to call out, not just slip in in what looks like an isolated change.

File: src/pivotal-ui/components/pui-variables.scss (hunk 1 above)
Comment (aredridel) [inline src/pivotal-ui/components/pui-variables.scss]: \o/

**Pull request merged**